google-osconfig-agent.service

```diff
@@ -9,6 +9,8 @@ Restart=always
 RestartSec=1
 StartLimitInterval=120
 StartLimitBurst=3
+KillMode=mixed
+KillSignal=SIGTERM
```

Comment:
this is the default for `KillMode=mixed`, right?

Reply:
Yes, I put it in here to make it clear what signal is getting sent without having to dig through docs first.